README.md

```diff
@@ -174,11 +174,12 @@ The `SECRET_KEY_BASE` and `MYSQL_PASSWORD` variables should be changed at the ve
 
 **Twilio/Authy Environment Variables**
 The following environment variables need to be set on the enrollment instances, which are the instances that will be dispatching the SMS/Voice assessments via Twilio and performing Two-Factor Authentication using Authy. These environment variables can be set in a `config/local_env.yml` file, or via a method provided by the deployment environment.
-* `TWILLIO_API_ACCOUNT: <Account number for Twilio Account>`
-* `TWILLIO_API_KEY: <API key for Twilio Account>`
-* `TWILLIO_SENDING_NUMBER: <Phone number registered to Twilio Account for SMS/Voice>`
-* `TWILLIO_STUDIO_FLOW: <Twilio Studio Flow ID for handling SMS/Voice Assessments>`
+* `TWILIO_API_ACCOUNT: <Account number for Twilio Account>`
```

Comment:
It's unfortunate the variable names were spelled wrong, but I'm concerned about changing these in the middle of this pandemic. Others besides us are hosting this, and this is a breaking change. Understand the need for `TWILIO_MESSAGING_SERVICE_SID`, but at least that is just one change. I think we need to support backwards on this. The `from` below should still work if `messaging_service_sid` is nil.